rest/data/host_create.go

```diff
@@ -192,12 +192,15 @@ func makeProjectAndExpansionsFromTask(t *task.Task) (*model.Project, *util.Expan
 	if err != nil {
 		return nil, nil, errors.Wrap(err, "error populating expansions")
 	}
-	params := append(proj.GetParameters(), v.Parameters...)
+	var params []patch.Parameter
```

Comment:
If projectInfo.Project == nil, we do need params to still equal v.Params, so maybe we should assign that here